ci-operator/step-registry/openshift/image-based/infra/aws/ec2/logs/openshift-image-based-infra-aws-ec2-logs-commands.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -xeuo pipefail
+set -euo pipefail
```

Comment:
Is this intended? This will disable logging which shell commands are executed and this seems like a generic script that should not be touched.

Reply:
ack, will it hurt us to not know which commands are run? Im guessing not, just making sure